bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1636,4 +1642,23 @@ private void postApprovalNeededComment() {
         pr.addComment(message);
     }
 
+    /**
+     * Iterate through each entry in the regularIssuesMap. If the value is present, get all CSR links associated with this issue.
+     * "all CSR links" means that the CSR linked with the primary issue and the CSRs linked with any backport issues of the primary issue.
+     * Constructs a map from main issue ID to all CSR links.
```

Comment:
I would like to comment to be more about what the return value is, not how it's created.
```suggestion
     * Creates a map from issue ID to a list of all CSRs linked from the issue or any backport of the issue.
```